src/UnityExtension/Assets/Editor/GitHub.Unity/UI/HistoryView.cs

```diff
@@ -207,7 +207,8 @@ private void RefreshLog()
             {
                 if (Repository != null)
                 {
-                    GitClient.Log().ThenInUI((success, log) => {
+                    GitClient.Log().ThenInUI((success, log) =>
+                    {
```

Comment:
I think you accidentally re-formatted this chunk of code here, could you restore it?